app/views/foreman/job_templates/install_errata.erb

```diff
@@ -15,4 +15,15 @@ foreign_input_sets:
   exclude: action,package
 %>
 
-<%= render_template('Package Action - SSH Default', :action => 'update', :package => "--advisories=#{input(:errata)}") %>
+<%
+  selector = if @host.operatingsystem && @host.operatingsystem.family == 'Redhat' && !@host.operatingsystem.major.blank?
+               if @host.operatingsystem.major.to_i >= 7
+                 '--advisories'
+               else
+                 '--advisory'
+               end
+             else
+               render_error(N_('Unsupported or no operating system found for this host.'))
```

Comment:
also it looks like --advisory still works on el7, would it make sense to just use that in all cases?